src/main/java/org/kohsuke/github/GitCommit.java

```diff
@@ -0,0 +1,211 @@
+        this.parents = commit.getParents();
+    }
+
+    /**
+     * Gets owner.
+     *
+     * @return the repository that contains the commit.
+     */
+    @SuppressFBWarnings(value = { "EI_EXPOSE_REP" }, justification = "Expected behavior")
+    public GHRepository getOwner() {
+        return owner;
+    }
+
+    /**
+     * Gets SHA1.
+     *
+     * @return The SHA1 of this commit
+     */
+    public String getSHA1() {
+        return sha;
+    }
+
+    /**
+     * Gets SHA.
+     *
+     * @return The SHA of this commit
+     */
+    public String getSha() {
+        return sha;
+    }
+
+    /**
+     * Gets node id.
+     *
+     * @return The node id of this commit
+     */
+    public String getNodeId() {
+        return node_id;
+    }
+
+    /**
+     * Gets URL.
+     *
+     * @return The URL of this commit
+     */
+    public String getUrl() {
+        return url;
+    }
+
+    /**
+     * Gets HTML URL.
+     *
+     * @return The HTML URL of this commit
+     */
+    public String getHtmlUrl() {
+        return html_url;
+    }
+
+    /**
+     * Gets author.
+     *
+     * @return the author
+     */
+    @WithBridgeMethods(value = GHCommit.GHAuthor.class, castRequired = true)
+    public GitUser getAuthor() {
+        return author;
+    }
+
+    /**
+     * Gets authored date.
+     *
+     * @return the authored date
+     */
+    public Date getAuthoredDate() {
+        return author.getDate();
+    }
+
+    /**
+     * Gets committer.
+     *
+     * @return the committer
+     */
+    @WithBridgeMethods(value = GHCommit.GHAuthor.class, castRequired = true)
+    public GitUser getCommitter() {
+        return committer;
+    }
+
+    /**
+     * Gets commit date.
+     *
+     * @return the commit date
+     */
+    public Date getCommitDate() {
+        return committer.getDate();
+    }
+
+    /**
+     * Gets message.
+     *
+     * @return Commit message.
+     */
+    public String getMessage() {
+        return message;
+    }
+
+    /**
+     * Gets Verification Status.
+     *
+     * @return the Verification status
+     */
+    public GHVerification getVerification() {
+        return verification;
+    }
+
+    Tree getTree() {
+        return tree;
+    }
+
+    public String getTreeSHA1() {
+        return tree.getSha();
+    }
+
+    public String getTreeUrl() {
+        return tree.getUrl();
+    }
+
+    @SuppressFBWarnings(value = "EI_EXPOSE_REP", justification = "acceptable")
+    List<GHCommit.Parent> getParents() {
+        return parents;
+    }
+
+    public List<String> getParentSHA1s() {
+        if (parents == null)
+            return Collections.emptyList();
+        return new AbstractList<String>() {
+            @Override
+            public String get(int index) {
+                return parents.get(index).sha;
+            }
+
+            @Override
+            public int size() {
+                return parents.size();
+            }
+        };
```

Comment:
This is `org.kohsuke.github.GitCommit.1`.

Reply:
oh I see - the message is pointing to the method, not the whole class. That makes a lot more sense to me, thank you!